### PR TITLE
Added this-month-rocks

### PR DESCRIPTION
**Pull request by koaning**
This is a website that I've made to inspire folks to do something fun this month. Made during the covid-19 pandemic for folks who are at home and may appreciate some inspiration. The website is fully collaborative, folks can add ideas via github.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -319,6 +319,7 @@
 - [Litekart](https://www.litekart.in)
 - [NuxtJS Documentation](https://nuxtjs.org)
 - [Vaggelis Yfantis](https://octoper.me)
+- [thismonthrocks](https://thismonth.rocks)
```
Comment (joshmanders) [inline readme.md:322]: Can you format the name better like maybe `This Month Rocks`

**Comment by joshmanders** [comment]
Also can you rebase with `master` so the new linting runs

File: readme.md (hunk 1 above)
Comment (koaning) [reply readme.md]: Fixed.

**Comment by koaning** [comment]
This should be it :)

**Comment by joshmanders** [comment]
Thanks @koaning!

**Pull request merged**